packages/phisherman/src/lib/Phisherman.ts

```diff
@@ -0,0 +1,90 @@
+import fetch, { Headers } from 'node-fetch';
```

Comment:
How do you manage to add a dependency on @sapphire/fetch but not use @sapphire/fetch lol